runtime/kusama/src/weights/runtime_common_auctions.rs

```diff
@@ -42,23 +43,48 @@ use sp_std::marker::PhantomData;
 /// Weight functions for `runtime_common::auctions`.
 pub struct WeightInfo<T>(PhantomData<T>);
 impl<T: frame_system::Config> runtime_common::auctions::WeightInfo for WeightInfo<T> {
+	// Storage: Auctions AuctionInfo (r:1 w:1)
+	// Storage: Auctions AuctionCounter (r:1 w:1)
 	fn new_auction() -> Weight {
-		(24_436_000 as Weight)
+		(11_899_000 as Weight)
```

Comment:
As explanation here and in other places: https://github.com/paritytech/substrate/pull/10692/ should have reduced all weights across all runtimes.  
Substrate weights were updated in that MR but Polkadot et.al. not.